lib/api/storage.ts

```diff
@@ -1,5 +1,7 @@
 export class ShredderStorage {
 
+  static storageKey = `_shredder`;
```

Comment:
@ilyagelman I don't think we need to prefix by location origin, localStorage is scoped by origin any way.